### ISSUE TITLE
Apollo-ios-cli in the root of the project via CI

### ISSUE THREAD

**Issue by justnicke** [apollographql/apollo-ios#3054]
### Question

Hi, I migrated from 0.43.0 to 1.0.7 version, everything is very convenient and stable through SPM, you have done a great job.  I still have a question related to setting up a build machine on CI

I have an understanding that with CI build job, the build machine should put the **apollo-ios-cli** at the root of the project, and then the commands are run sequentially in the build phase of the project **./apollo-ios-cli fetch-schema** and **./apollo-ios-cli generate.**

The question is how can I put the **apollo-ios-cli** in the root of the project for the build machine?

**Comment by calvincestari** [comment]
@justnicke, dependency resolution for both SPM and CocoaPods should create a symbolic link for you already in the root folder of your project. Having an Xcode project that pulls in Apollo iOS via SPM is where it gets tricky for CI workflow. There is the 'Install CLI' plugin but it's not very suitable for headless CI workflows.

You can build the CLI from the command line with the makefile command `make build-cli` but that requires you to know where Xcode has put the Apollo iOS dependency. Alternatively you can download the CLI directly from GitHub for the latest release using the command `curl -LO https://github.com/apollographql/apollo-ios/releases/latest/download/apollo-ios-cli.tar.gz` but you'll need to be aware of version releases and ensure you downloading a version that matches your dependency version.

tl;dr - managing dependencies outside of Xcode is the easiest way to do this on CI.
